src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/Radioactivity.java

```diff
@@ -20,39 +20,43 @@
      * This represents a low level of radiation.
      * It will still cause damage but will take a while before it becomes deadly.
      */
-    LOW(ChatColor.YELLOW),
+    LOW(ChatColor.YELLOW, 1),
 
     /**
      * This represents a medium level of radiation.
      * This can be considered the default.
      */
-    MODERATE(ChatColor.YELLOW),
+    MODERATE(ChatColor.YELLOW, 2),
 
     /**
      * This is a high level of radiation.
      * It will cause death if the {@link Player} does not act quickly.
      */
-    HIGH(ChatColor.DARK_GREEN),
+    HIGH(ChatColor.DARK_GREEN, 3),
 
     /**
      * A very high level of radiation will be deadly.
      * The {@link Player} should not take this too lightly...
      */
-    VERY_HIGH(ChatColor.RED),
+    VERY_HIGH(ChatColor.RED, 5),
 
     /**
-     * This is the deadlies level of radiation.
+     * This is the deadliest level of radiation.
      * The {@link Player} has basically no chance to protect themselves in time.
      * It will cause certain death.
      */
-    VERY_DEADLY(ChatColor.DARK_RED);
+    VERY_DEADLY(ChatColor.DARK_RED, 10);
 
     private final ChatColor color;
-
-    Radioactivity(@Nonnull ChatColor color) {
+    private final int exposureModifier;
+    Radioactivity(@Nonnull ChatColor color, int exposureModifier) {
         this.color = color;
+        this.exposureModifier = exposureModifier;
+    }
+    
+    public int getExposureModifier() {
+        return exposureModifier;
     }
```

Comment:
This is missing some javadocs, when you introduce a new attribute like this here then please document what this represents.